prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -517,14 +518,26 @@ class ActionHistory(
         filename: String?,
         params: String,
         result: String,
-        itemCount: Int
+        header: WorkflowEngine.Header
     ) {
         if (isReportAlreadyTracked(sentReportId)) {
             error(
                 "Bug:  attempt to track history of a report ($sentReportId) " +
                     "we've already associated with this action"
             )
         }
+
+        if (header.content == null) {
+            error("Bug: attempt to track sent report with no contents")
```

Comment:
This should not be possible as the contents here are just what was sent to the receiver, but `content` is nullable.